src/components/AddCard/AddCard.test.a11y.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+/* eslint-disable */
+import { mount } from 'enzyme';
+import AAT from '@ibma/aat';
+
+import HTMLWrap from '../../a11y/a11y-test-helper.jsx';
+
+import AddCard from './AddCard.jsx';
+/* eslint-enable */
```

Comment:
What does eslint bark about here with these imports?

Reply:
It doesn't want me to use the jsx extension but since these are in the same directory and not index.jsx I need to add it

Reply:
I removed the extensions as they actually were not a problem. It is still barking at me to move enzyme to a dependency as opposed to dev dependency.

Reply:
This is caused by this file ending in `.a11y.jsx` as opposed to `.test.jsx`